src/node/txreconciliation.cpp

```diff
@@ -97,6 +97,13 @@ class TxReconciliationTracker::Impl
      */
     std::unordered_map<NodeId, std::variant<uint64_t, TxReconciliationState>> m_states GUARDED_BY(m_txreconciliation_mutex);
 
+    /**
+     * Maintains a queue of reconciliations we should initiate. To achieve higher bandwidth
+     * conservation and avoid overflows, we should reconcile in the same order, because then it’s
```

Comment:
The comment could be more precised and if I’m understanding correctly it could say the order of reconciliating peers should be maintained between each reconciliation.